tests/creation/create.act.typed.json

```diff
@@ -1 +1,97 @@
-[]
+[
+  {
+    "kind": "Invariants",
+    "contracts": [
+      "Modest"
+    ],
+    "invariants": []
+  },
+  {
+    "stateUpdates": {
+      "x": [
+        {
+          "Right": {
+            "location": "x",
+            "value": 1
+          }
+        }
+      ],
+      "y": [
+        {
+          "Right": {
+            "location": "y",
+            "value": "Caller"
+          }
+        }
+      ]
+    },
+    "kind": "Behaviour",
+    "mode": "Pass",
+    "creation": true,
+    "contracts": [
+      "Modest"
+    ],
+    "name": "creation",
+    "preConditions": "True",
+    "contract": "Modest",
+    "interface": "constructor()",
+    "postConditions": {
+      "arity": 2,
+      "args": [
+        {
+          "arity": 2,
+          "args": [
+            {
+              "arity": 2,
+              "args": [
+                0,
+                "x"
+              ],
+              "symbol": "<="
+            },
+            {
+              "arity": 2,
+              "args": [
+                "x",
+                1.157920892373162e+77
```

Comment:
hmm, we should probably print this as a string to avoid javascript doing nasty things to our numbers